applications_tests/lethe-fluid/mortar_taylorcouette.output

```diff
@@ -0,0 +1,34 @@
+Running on 1 MPI rank(s)...
```

Comment:
If this work in parallel, could you also make sure this test is run in parallel ?